save/20176.0.txt

```diff
@@ -0,0 +1,47 @@
+@0
+
+
+@
+#this code puts the value into the "acc" function and subtracts it from the value coming from the right and moves that value down to the "out" and then it does the opposite
```

Comment:
Careful of terminology -- `ACC` is a register (similar to a variable), not a function. It's worth being precise with terminology like this to avoid confusion.